keyvi/src/cpp/dictionary/dictionary_merger.h

```diff
@@ -87,105 +91,124 @@ final {
 
     private:
         EntryIteratorPtr entry_iterator_ptr_;
-        int priority_;
+        size_t segmentIndex_;
     };
 
- public:
-  DictionaryMerger(size_t memory_limit = 1073741824,
-                   const merger_param_t& params = merger_param_t()):
-                     dicts_to_merge_(), memory_limit_(memory_limit), params_(params){
-    if (params_.count(TEMPORARY_PATH_KEY) == 0) {
-      params_[TEMPORARY_PATH_KEY] = boost::filesystem::temp_directory_path().string();
+public:
+    DictionaryMerger(bool append_merge = false,
+                     size_t memory_limit = 1073741824, const merger_param_t& params = merger_param_t())
```

Comment:
good point, what about doing that for the 0.2 release? But we have to do that in other parts as well then.